src/babashka/cli.cljc

```diff
@@ -321,6 +325,19 @@
                     (-> {:spec spec :type :org.babashka/cli}
                         (merge data)
                         error-fn*))
+         [opt shell cmdline] args
+         _ (case opt
+             "--babashka.cli/completion-snippet"
+             (if-let [command-name (get-in opts [:completion :command])]
+               (do (print-completion-shell-snippet (keyword shell) command-name)
+                   (System/exit 0))
+               (binding [*out* *err*]
+                 (println "Need `:completion {:command \"<name>\"}` in opts to support shell completions")
+                 (System/exit 1)))
+             "--babashka.cli/complete"
```

Comment:
Let's go with `org.babashka/cli` similar to how metadata works in bb.cli: everything under that "domain" is reserved

Reply:
Sorry, I mean `--org.babashka.cli/....`